compiler/msgs.nim

```diff
@@ -907,3 +960,6 @@ ropes.errorHandler = proc (err: RopesError, msg: string, useWarning: bool) =
   of rCannotOpenFile:
     rawMessage(if useWarning: warnCannotOpenFile else: errCannotOpenFile, msg)
 
+# enable colors by default on terminals
+if terminal.isatty(stdout):
```

Comment:
Or when some process is consuming Nim output and outputting sting else, then calling colors API could mess output on Windows.